packages/connect-wallet-modal/package.json

```diff
@@ -48,6 +48,7 @@
     "@reef-knot/wallets-helpers": "^1.1.5",
     "@reef-knot/web3-react": "^1.4.2",
     "@types/ua-parser-js": "^0.7.36",
+    "eslint-config-custom": "*",
```

Comment:
Why is it "*"?

Reply:
This is an internal package placed in this repo and has no actual version,
also see these instructions: https://turbo.build/repo/docs/handbook/linting/eslint#how-to-use-the-eslint-config-custom-package